README.md

```diff
@@ -2,28 +2,32 @@
 
 ## Installation
 
+### Production Install
+
 Requirements: python3, postgresql-10
 
 Set up a venv and install requirements:
 ```
 virtualenv -p python3 .ve
 source .ve/bin/activate
 pip install -r requirements.txt
+pip install -e .
 ```
 
 Example of creating an ocdsdata table and data:
 ```
 sudo -u postgres createuser ocdsdata --pwprompt
-sudo -u postgres createdb ocdsdata -O ocdsdata --encoding UTF8 --template template0 --lc-collate en_US.UTF-8 --lc-ctype en_US.UTF-8 
+sudo -u postgres createdb ocdsdata -O ocdsdata --encoding UTF8 --template template0 --lc-collate en_US.UTF-8 --lc-ctype en_US.UTF-8
 export DB_URI='postgres://ocdsdata:PASSWORD YOU CHOSE@localhost/ocdsdata'
 python ocdsdata-cli upgrade-database
 ```
 
+=======
```

Comment:
This is an artifact of the Git conflict I think? I don't think this should be here